cms/static/js/views/assets.js

```diff
@@ -243,9 +266,11 @@ define(["jquery", "underscore", "gettext", "js/views/baseview", "js/models/asset
             },
 
             showUploadFeedback: function (event, percentComplete) {
-                var percentVal = percentComplete + '%';
-                $('.upload-modal .progress-fill').width(percentVal);
-                $('.upload-modal .progress-fill').html(percentVal);
+                var percentVal = percentComplete + '%',
+                    $progressFill = $('.upload-modal .progress-fill');
+
+                $progressFill.width(percentVal);
+                HtmlUtils.setHtml($progressFill, percentVal);
```

Comment:
If the value is not actually HTML, you can just use this:

    $progessFill.text(percentVal);